spec/controllers/followings_controller_spec.rb

```diff
@@ -24,6 +24,118 @@
       expect(assigns(:followings)).to eq(alice.followings)
     end
 
-    it 'redirects to the singin path if the user is not authenticated'
+    it 'orders the followings such that last followings shows first' do
+      jose = Fabricate(:user)
+      carlos = Fabricate(:user)
+      following1 = Fabricate(:following, user_id: alice.id, subject_id: jose.id)
+      following2 = Fabricate(:following, user_id: alice.id, subject_id: carlos.id)
+      get :index, user_id: alice.id
+      expect(assigns(:followings)).to match_array([following2, following1])
```

Comment:
`match_array` does not check order, only that the elements exist. The description of this spec does not match this expectation.